docs/guides/Guide-Intro.md

```diff
@@ -51,10 +51,11 @@ class HomeScreen extends React.Component {
   }
 }
 
-const SimpleApp = StackNavigator({
+export default SimpleApp = StackNavigator({
   Home: { screen: HomeScreen },
 });
 
+// skip this line if using Create React Native App
```

Comment:
I think it might be best to have two "Hello World" examples for the absolute beginner. There was a [recent issue](https://github.com/react-community/react-navigation/issues/1945) where someone could not get the Hello World example running by copying and pasting this example into their text editor. I am in favor of having two code snippets: one for the `react-native` CLI and another tailored for `create-react-native-app` with a brief explanation _why_ the two snippets are different. A lot of beginners jump straight into using libraries like this one, and we want to minimize the amount of potential confusion that they might have.